jobs/config.json

```diff
@@ -8498,6 +8497,22 @@
       "sig-aws"
     ]
   },
+  "ci-kubernetes-e2e-kops-aws-latest": {
```

Comment:
this is same as ci-kubernetes-e2e-kops-aws?

Reply:
It is - good spot.  I'll remove it.  I need to do a pass on all these jobs to make sure we don't have dups (and maybe name them following a pattern).  And I also need to move this job to the new runner (almost there!)